main.go

```diff
@@ -10,7 +10,8 @@ import (
 func main() {
 	myRouter := mux.NewRouter().StrictSlash(true)
 	deviceService := deviceHTTPHandler{service: DeviceService{dao: &inMemoryDeviceDAO{}}}
-	myRouter.HandleFunc("/devices", deviceService.createDevice).Methods("POST")
+	myRouter.HandleFunc("/devices/", deviceService.createDevice).Methods(http.MethodPost)
```

Comment:
This endpoint should be `POST /devices` but is `POST /devices/` - the trailing slash should not be required.